lib/Serge/Engine/Plugin/parse_yaml.pm

```diff
@@ -26,6 +26,8 @@ sub init {
         email_from     => 'STRING',
         email_to       => 'ARRAY',
         email_subject  => 'STRING',
+
+        yaml_mode      => 'STRING',
```

Comment:
In that case I think `yaml_type` is more natural than `yaml_kind`, but will name it `yaml_kind` for consistency.